rdt-open-reader/src/covidIndonesia/assets/json.form/sample-collection-form-original.json

```diff
@@ -816,20 +816,50 @@
         "text": "Please confirm the information"
       },
       {
-        "key": "patient_detail",
+        "key": "patient_info_unique_id",
```

Comment:
This can be named `unique_id` allowing the already-existing logic which pre-populates the other unique id fields to kick in.